packages/web-components/src/button/button.styles.ts

```diff
@@ -308,4 +308,11 @@ export const styles = css`
     color: ${colorNeutralForegroundDisabled};
     cursor: not-allowed;
   }
+
+  @media (forced-colors: active) {
+    :host(:hover) .control,
+    :host([appearance='transparent']:hover) .control {
+      border-color: Highlight;
+    }
+  }
```

Comment:
forcedColorStyelsheet? Also, it's a bit odd that this is applied to _every_ instance and then also a specific instance via selectors...